sdf.md

```diff
@@ -827,6 +827,18 @@ definition specify that the
 interactions offered by a Thing modeled by this sdfThing include the
 interactions modeled by the nested `sdfObject` or `sdfThing`.
 
+## Top-level Affordances and sdfData
+
+Besides their placement within an `sdfObject` or `sdfThing`, both affordances
+(i.e., `sdfProperty`, `sdfAction`, and `sdfEvent`) as well as `sdfData` can
+also be placed at the top level of an SDF model.
+Since they are not associated with an `sdfObject` or `sdfThing`, these kinds of
+definitions are primilary intended to be re-used via the sdfRef mechanism
+(see {{sdfref}}).
+<!-- TODO: Should this be possible? Or is it discouraged? -->
```

Comment:
This may be opening up a can of worms so keeping the text just on re-use seems reasonable to me

Reply:
Thank you for feedback, @akeranen! I removed the line in [4c2006e](https://github.com/ietf-wg-asdf/SDF/pull/141/commits/4c2006e88c98b4b5e76292eee1f3470685a4a036) – leaving it out also sounds reasonable to me :)

I guess the YANG mapping needs to be revisited at some point, though, to take this decision into account, and/or an additional way of describing this kind of action needs to added via an extension mechanism later.